### ISSUE TITLE
Did the baselines use the same (CLIP + SlowFast) feature extractors as Moment-DETR?

### ISSUE THREAD

**Issue by yeliudev** [jayleicn/moment_detr#46]
Hi @jayleicn, many thanks for sharing this great work! I was wondering whether your baseline models (e.g., MCN, XML, XML+) in Table 3 used the same feature extractors as Moment-DETR? Thanks!

**Comment by jayleicn** [comment]
Yes, we used the same features to these models to ensure fair comparisons.

**Comment by yeliudev** [comment]
I see. Thanks for your reply!
